e2e/test/scenarios/dashboard-filters/old-parameters.cy.spec.js

```diff
@@ -87,15 +95,13 @@ describe("scenarios > dashboard > OLD parameters", () => {
       rightSidebar().findByLabelText("Auto-apply filters").click();
       cy.button("Apply").should("not.exist");
 
-      cy.button("Category").click();
-      popover().within(() => {
-        cy.findByText("Gadget").click();
-        cy.findByText("Add filter").click();
+      filterWidget().within(() => {
```

Comment:
I need to modify this existing test a little because previously the parameter values are clear after toggling auto-apply filters off, but not anymore.

So I change from adding a filter to removing a filter instead since the first section of the test already add a filter.